packages/x-charts/src/ChartsTooltip/ChartsTooltip.tsx

```diff
@@ -1,8 +1,9 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import { unstable_composeClasses as composeClasses } from '@mui/utils';
-import { Popper } from '@mui/base/Popper';
+import Popper, { PopperProps } from '@mui/material/Popper';
```

Comment:
Was there any problem with the `base` Popper when doing this change? 🤔

Reply:
Thanks for the catch. I tried to use the new [Popup](https://mui.com/base-ui/react-popup/) but I reverted to material instead of base